zebra/zebra_vxlan.c

```diff
@@ -4668,7 +4668,7 @@ int zebra_vxlan_if_up(struct interface *ifp)
 			zevpn->vrf_id = vlan_if->vrf_id;
 			zl3vni = zl3vni_from_vrf(vlan_if->vrf_id);
 			if (zl3vni)
-				listnode_add_sort(zl3vni->l2vnis, zevpn);
+				listnode_add_sort_nodup(zl3vni->l2vnis, zevpn);
```

Comment:
whilst this works, isn't the real probelm that the corresponding listnode_delete processing is missing from `zebra_vxlan_if_down`